eng/native/init-distro-rid.sh

```diff
@@ -41,8 +41,8 @@ initNonPortableDistroRid()
             # We have forced __PortableBuild=0. This is because -portablebuld
             # has been passed as false.
             if (( isPortable == 0 )); then
-                if [[ "${ID}" == "rhel" || "${ID}" == "rocky" ]]; then
-                    # remove the last version digit
+                # for RHEL and its derivatives, remove the last version digit (if present)
+                if [[ "${ID}" == "rhel" || "${ID_LIKE}" =~ .*rhel.* ]]; then
```

Comment:
does this also affect centos and fedora?

Reply:
The condition matches centos but not fedora.
After that value on centos is not affected by the line below, so it is only for distros which we have version / release policy like RHEL and are derivatives of that.

Reply:
Also, these PRs are not really needed, unless the distro really wants to create a "non-portable" release of .NET and provide some distro-specific packages.

https://dotnet.microsoft.com/download/dotnet/thank-you/sdk-6.0.100-linux-x64-binaries works for any glibc-based linux distro..

`non-portable` is only used by source builds for a long time now, elsewhere we all use portable binaries, so these PRs are less meaningful.